gateways/http/account/account.go

```diff
@@ -33,17 +34,25 @@ func Accounts(serv *mux.Router, usecase account.UseCase) *Handler {
 // @Header 201 {string} Token "request-id"
 // @Router /accounts [post]
 func (h Handler) CreateAccount(w http.ResponseWriter, r *http.Request) {
-	var acc entities.AccountInput
+	var accountInput entities.AccountInput
 	decoder := json.NewDecoder(r.Body)
-	err := decoder.Decode(&acc)
+	err := decoder.Decode(&accountInput)
 	if err != nil {
-		//implenmentar erro aqui
+		w.WriteHeader(http.StatusBadRequest)
```

Comment:
what do you think to send an informative message to tell your user what went wrong when there's an error? besides, you're forgetting to return after setting the response status code

Reply:
All application error responses I plan to adjust in the next branch.